core/src/main/java/org/apache/carbondata/core/scan/filter/executer/IncludeFilterExecuterImpl.java

```diff
@@ -631,4 +628,99 @@ public void readColumnChunks(RawBlockletColumnChunks rawBlockletColumnChunks) th
       }
     }
   }
+
+  /**
+   * Below method will be used to execute measure filter based on data type
+   * This is done to avoid conversion of primitive type to primitive object
+   * as it may cause lots of gc when number of record is high and will impact performance
+   * @param page
+   * @param numberOfRows
+   * @param bitSet
+   * @param nullBitset
+   * @param measureColumnExecuterFilterInfo
+   * @param dataType
+   */
+  private void executeIncludeFilterMeasure(ColumnPage page, int numberOfRows, BitSet bitSet,
```

Comment:
Its almost a duplicate method to `executeExcludeFilterForMeasure`. Better pass a small wrapper object of Bitset to this method to flip/set the bit and use the same method for both of them.